src/roles/common/tasks/fix-image-paths.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Get unzipped files dir
+  set_fact:
+    nuage_unzipped_files_dir: "{{ common.nuage_unzipped_files_dir }}"
+
+- name: Replace non-relative path

```

Comment:
This and the below task should only be performed when inside the container (check if role path starts with /source/nuage-metroae)

Reply:
what is the role path? I'm assuming that's different from the path provided by common.nuage_unzipped_files_dir? Currently, I think the role path is the path the fix-image-paths.yml file would belong to (the path provided if a pwd command were executed within the directory)?

Reply:
There is a special variable set by Ansible: {{ role_path }}